moto/iam/models.py

```diff
@@ -119,10 +119,11 @@ def __init__(self, role_id, name, assume_role_policy_document, path):
     def create_from_cloudformation_json(cls, resource_name, cloudformation_json, region_name):
         properties = cloudformation_json['Properties']
 
+        path = properties['Path'] if 'Path' in properties else '/'
         role = iam_backend.create_role(
             role_name=resource_name,
             assume_role_policy_document=properties['AssumeRolePolicyDocument'],
-            path=properties['Path'],
```

Comment:
Replace with `properties.get('Path', '/')`

Reply:
Doi yeah that's way smarter.